docs/gaia/validators/validator-setup.md

```diff
@@ -53,20 +53,19 @@ __Note__: If unspecified, `consensus_pubkey` will default to the output of `gaia
 
 ## Participate in genesis as a validator
 
-__Note__: This section only concerns validators that want to be in the genesis file. If the chain you want to validate is already live, skip this section.
+__Note__: This section only concerns validators that want to be in the genesis
+file. If the chain you want to validate is already live, skip this section.
 
-__Note__: `Gaia-9002` and `Game of stakes` will not use this process. They will be bootsrapped using Tendermint seed validators. You will just need to use the [create-validator](#create-your-validator) command in order to join as a validator for these networks.
+__Note__: `Gaia-9002` and `Game of stakes` will not use this process. They will
+be bootstrapped using Tendermint seed validators. You will just need to use the
+[create-validator](#create-your-validator) command in order to join as a validator
+for these networks.
 
-If you want to participate in genesis as a validator, you need to justify that you (or a delegator) have some stake at genesis, create one (or multiple) transaction to bond this stake to your validator address, and include this transaction in the genesis file. 
+If you want to participate in genesis as a validator, you need to justify that
+you have some stake at genesis, create one (or multiple) transaction to bond this
```

Comment:
```suggestion
you have some stake at genesis, create one (or multiple) transactions to bond this
```

Can you create multiple - do we allow that? I don't think so - not with the same operator address.